tests/test_channel_creation.py

```diff
@@ -33,7 +34,7 @@ class TestAnalogCreateChannels(object):
     """
 
     @pytest.mark.parametrize("seed", [generate_random_seed()])
-    def test_create_ai_voltage_chan(self, any_x_series_device, seed):
+    def test_create_ai_voltage_chan(self, any_x_series_device, seed):  # noqa: F811
```

Comment:
[F811: Redefinition of unused name from line n ](https://www.flake8rules.com/rules/F811.html) seems wrong. What is the error?

Reply:
This is everywhere, I'll only mark it once :D

Reply:
It's complaining since `any_x_series_device` is imported, but then redefined as a parameter here.

Note on how Pytest works: the fixtures don't need to be imported into the test files.